src/corecel/Macros.hh

```diff
@@ -173,3 +173,22 @@
 #else
 #    define CELER_DEVICE_PREFIX(TOK) DEVICE_UNAVAILABLE
 #endif
+
+/*!
+ * \def CELER_TRY_ELSE
+ *
+ * Execute the statement, catching *all* thrown errors by calling the given
+ * function-like operator with a \c std::exception_ptr object.
+ *
+ * \note A file that uses this macro must include the <exception> header (but
+ * since the \c HANDLE_EXCEPTION needs to take an exception pointer anyway).
```

Comment:
…it’s probably included?

Reply:
Yep, I tried to revise that sentence to make it easier to understand and ended up leaving half of it out 😂